src/db/migration.003.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+-- create temporary table
+CREATE TABLE tempmeasurement(
+    runid smallint NOT NULL,
+    trialid smallint NOT NULL,
+    criterion smallint NOT NULL,
+    invocation smallint NOT NULL,
+    value float4[] NOT NULL,
+    PRIMARY KEY(runid,trialid,criterion,invocation)
+);
+-- order the measurement table by primary key, then insert into the new temporary table the aggregate of value for iteration
+INSERT INTO tempmeasurement(runid,trialid,criterion,invocation,value)
+WITH iterations AS(SELECT runid,trialid,criterion,invocation,iteration, value FROM measurement ORDER BY runid,trialid,criterion,invocation,iteration ASC)
+SELECT runid,trialid,criterion,invocation, array_agg(value) as aggValue FROM iterations GROUP BY runid,trialid,criterion,invocation ;
```

Comment:
could you find a better name than `aggValue`? What is this? The array of values? how about `arrayOfValues`?

It's also impossible for me to properly read this query and it's complex use of subqueries. Please reformat.

Reply:
aggregateValues

Reply:
`aggregatedValues` does not resonate with me. A sum is also a type of aggregation.

Is there anything wrong with `arrayOfValues`? It says that it's an array, and that it contains the values.

Reply:
array_agg is aggregateValues but the reuslt is an arrayOfValues. so arrayOfValues would be better